scripts/cmake-presets/.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
If this file is still giving you trouble, let me know and I will try to delete it on your branch.